monkestation/code/modules/virology/disease/symtoms/stage2.dm

```diff
@@ -518,26 +487,21 @@
 	badness = EFFECT_DANGER_HINDRANCE
 	var/active = 0
 
-/datum/symptom/cult_vomit/activate(mob/living/carbon/M)
-	if(!ishuman(M) || active)
+/datum/symptom/cult_vomit/activate(mob/living/carbon/mob)
+	if(!ishuman(mob) || active)
 		return
-	if(istype(get_area(M), /area/station/service/chapel))
+	if(istype(get_area(mob), /area/station/service/chapel))
 		return
-	if(IS_CULTIST(M))
+	if(IS_CULTIST(mob))
 		return
 
-	var/mob/living/carbon/human/mob = M
+	var/mob/living/carbon/human/victim = mob
 	active = 1
-	to_chat(mob, span_warning("You feel a burning sensation in your throat."))
+	to_chat(victim, span_warning("You feel a burning sensation in your throat."))
 	sleep(10 SECONDS)
-	to_chat(mob, span_danger("You feel an agonizing pain in your throat!"))
+	to_chat(victim, span_danger("You feel an agonizing pain in your throat!"))
 	sleep(10 SECONDS)
-	mob.visible_message(span_danger("[mob] vomits up blood!"), span_danger("You vomit up blood!"))
-	var/obj/effect/decal/cleanable/blood/S = new(loc = get_turf(mob))
```

Comment:
this shouldn't be removed so it can trigger on things with no hunger without causing massive issues